src/templates/bespoke/bespoke.scss

```diff
@@ -220,6 +223,17 @@ $progress-height: 5px;
         }
       }
 
+      // Dragbar
+      .bespoke-marp-presenter-dragbar-container {
+        grid-area: dragbar;
+        background: white;
```

Comment:
```suggestion
```

Reply:
maybe something slightly off black if you don't like a big white line?
or maybe a `:hover` state so people know it exists apart from the cursor changing?

Reply:
VS Code's splitter is a good one to reference.

https://user-images.githubusercontent.com/3993388/155305776-3e38b777-202f-47cc-a7fd-9b7e01f3ab14.mov

Normally there is not a visible line. If hovered the splitter, it would be highlighted by blue line after a bit of delay. This delay is important to avoid blinking the line whenever the pointer crosses over.

Reply:
Example:

```scss
.bespoke-marp-presenter-dragbar-container {
  background: #0288d1; // Marp brand color
  opacity: 0;
  transition: opacity 0.2s linear 0.4s;

  &:hover {
    opacity: 1;
  }
}
```

Reply:
fixed in 29e35c71